### ISSUE TITLE
One question about this bitch detection - Wacatac.H!ml

### ISSUE THREAD

**Issue by notevil1337** [UnamSanctam/SilentCryptoMiner#1855]
Hello Unam, have you ever noticed this detection type (Wacatac.H!ml)?
I've seen this form of detection before and noted the results (in quite a few .NET projects) but I can't remember what was causing it.
Also this detection is very common, if you know this can you tell me what causes such common detections from defender ?

**Comment by notevil1337** [comment]
Also this virus happens even when all build settings are disabled (except for one empty miner)

**Comment by UnamSanctam** [comment]
Yes that's just a general detection name, the `ml` part means that it was detected by machine learning. They use generalized names like `Wacatac` for a ton of different detections so that you won't know what it actually detected.

**Comment by notevil1337** [comment]
Bypassed every detections )

**Comment by notevil1337** [comment]
@UnamSanctam It shows 2.9 KH/s on the panel and on the pool
1.76 kH/s is a long time already and why

**Comment by UnamSanctam** [comment]
Sorry I don't understand your question.

**Comment by notevil1337** [comment]
![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/141959915/ff8133ca-1121-4520-9ec7-b99904702ba2)
and on the pool it displays 
![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/141959915/1ead63eb-c3ff-4869-baf1-a5572fcb9ef6)
![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/141959915/8e2d4ecf-f319-40fc-8db4-fd780acd9c03)

**Comment by notevil1337** [comment]
Do you understand now?

**Comment by notevil1337** [comment]
Pool displays lower hashrate than panel

**Comment by UnamSanctam** [comment]
Yes, 2.9 KH/s is the real hashrate that it's currently mining at. While the pool calculates the hashrate by the amount of shares that it submits over some interval of time.

**Comment by notevil1337** [comment]
@UnamSanctam I need your help really fast

**Comment by notevil1337** [comment]
![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/141959915/eb1dfe2c-791d-45e7-88fd-ca2b2a086744)
Its my old configuration and where it says wrong address I forgot to change it
![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/141959915/37209b0b-facb-49d3-a39a-c5271236b93f)

Please tell me how can i change the wallet and in general, when is this Pastebin link launched or what even is this?

**Comment by notevil1337** [comment]
@UnamSanctam I simply want to change the Remote URL
